config/index.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
   },
   dev: {
     env: require('./dev.env'),
-    port: 8080,
+    port: 8000,
```

Comment:
This is good practice. You can change it to either `3000` or leave it as now.

Reply:
okay